eth2-book-guiding-doc.md

```diff
@@ -0,0 +1,231 @@
+We're initially targeting the spectrum of **people with a basic knowledge of Ethereum and some interest in Eth2**.
+These readers may or may not have an understanding of the technical underpinnings of Ethereum.
+
+## Content Structure
+We want to develop new ways to present content in order to accommodate the widest possible audience.
+We're experimenting with a content structure that consists of a **knowledge base** and corresponding **educational experiences**. 
+
+### Knowledge Base
+The **knowledge base** component of this project is the core repository of knowledge.
+It's intended to be a **searchable** database of information, much like a wiki.
+Users should be able to use the base to find any piece of Eth2 information they might be looking for.
+
+### Educational Experiences
+Although large repositories of information are extremely useful, they tend to be inaccessible when a user doesn't already know what they're looking for.
+Users shouldn't have to sift through piles of information by themselves.
+For that reason, it's valuable to provide **educational experiences** that present information from the knowledge base in approachable ways.
+We're still working on the exact form that these experiences will take.
+However, because they rely on the knowledge base for any underlying information, these experiences can be developed once the knowledge base has been mostly completed.
+
+## Collaboration
+Much of the knowledge about Eth2 is "[tribal](https://en.wikipedia.org/wiki/Tribal_knowledge)."
+It's important that members of the Eth2 R&D community be able to contribute knowledge to our project.
+Contributors will be able to submit "raw" content from which an editing team will generate some final content.
+We're hoping this will reduce friction across the board.
+
+- - - -
+
+## Design Requirements
+
+### Website
+* Should be easily discoverable online.
+* Should use accessibility best-practices.
+* Should be accessible on any device.
+* Should take full advantage of the web platform.
+* Should be designed to improve the learning experience.
+
+### Language
+* Should always strive for simplicity.
+* Should avoid using confusing or overly technical terminology.
+* Should avoid condescension.
+
+### Content
+* Should have different entry points for different users.
+* Should be effectively linked to other content within the resource.
+
+### Projects
+* Should be fluidly integrated into the learning experience.
+* Should be easily accessible.
+
+- - - -
+## Outline
+
+### Examining Ethereum
+* Overview of Eth1
+	* Historical context
+	* Structure
+	* Limitations
+* Overview of scaling options
+	* Supernodes
+	* Sharding
+	* L2
+
+#### Sources
+* [Eth1 Design Rationale](https://github.com/ethereum/wiki/wiki/Design-Rationale)
+
+### Eth2 Overview
+* Overview of Eth2
+	* Historical context
+	* Eth2 Protocol
+		* Design rationale
+		* Principles
+		* How Eth2 addresses limitations of Eth1
+	* Protocol explanation
+		* PoS (Casper)
```

Comment:
Fork choice rule (LMD GHOST) is good to have in or right after the PoS section as they are intimately related